scapy/contrib/pnio.py

```diff
@@ -108,9 +108,15 @@ class ProfinetIO(Packet):
 
     def guess_payload_class(self, payload):
         # For frameID in the RT_CLASS_* range, use the RTC packet as payload
-        if (
+        if self.frameID == 0xfefe or 0xfeff or 0xfefd:
```

Comment:
That won't work in Python.
```suggestion
        if self.frameID in [0xfefe, 0xfeff, 0xfefd]:
```